src/SearchResults.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const SearchResults = ({ results }) => {
+  
+  const [selectedRows, setSelectedRows] = useState([]);
+  // create a new state variable 'selectedRows' using the useState hook.
+//  here will be store the selected rows 
+// we start from empty array
+
+  const handleRowClick = (id) => {
+    if (selectedRows.includes(id)) {
+      setSelectedRows(selectedRows.filter(rowId => rowId !== id));
+    } else {
+      setSelectedRows([...selectedRows, id]);
```

Comment:
This is a cool way of doing this. 
Can you think of another way where we just keep a `selected` state, which is equal to true or false?